django/core/checks/security/base.py

```diff
@@ -97,6 +108,19 @@
     id='security.W021',
 )
 
+W022 = Warning(
+    "You have not set the SECURE_REFERRER_POLICY setting. Without this, your "
+    "site will not send a Referrer-Policy header. You should consider enabling "
+    "this header to protect user privacy.",
+    id="security.W022"
+)
```

Comment:
Single quotes please and wrap at 119 characters. (Please do this everywhere as appropriate.)